modules/auxiliary/spoof/llmnr/llmnr_response.rb

```diff
@@ -176,7 +174,10 @@ def should_print_reply?(host)
 
   def run
     check_pcaprub_loaded()
-    ::Socket.do_not_reverse_lookup = true
+    ::Socket.do_not_reverse_lookup = true  # Mac OS X workaround
+
+    # Avoid receiving extraneous traffic on our send socket
+    open_pcap({'FILTER' => 'ether host f0:f0:f0:f0:f0:f0'})
```

Comment:
Any MAC that doesn't match normal traffic. The Capture mixin creates two Pcap interfaces, one for ARP and GW discovery, and another for sending responses. Prior to this change, every send would open_pcap, capture_sendto, and close_pcap, which was inefficient. The new code calls open_pcap at the start and close_pcap in cleanup, but that means any traffic hitting the default filter will start to get buffered. The bogus host filter prevents random traffic from queueing up on the send Pcap instance.